repository-meta-analyzer/src/test/java/org/hyades/repositories/NugetMetaAnalyzerTest.java

```diff
@@ -110,7 +110,7 @@ void testAnalyzer() throws Exception {
         Assertions.assertEquals(RepositoryType.NUGET, analyzer.supportedRepositoryType());
         Assertions.assertNotNull(metaModel.getComponent());
         Assertions.assertNotNull(metaModel.getLatestVersion());
-        Assertions.assertNotNull(metaModel.getPublishedTimestamp());
+        Assertions.assertNull(metaModel.getPublishedTimestamp());
```

Comment:
Is publishedTimestamp the timestamp of latest version ? In that case it should be Not Null

Reply:
the url that the test is going to is set to return 404 in which case the latest version of the component is the existing version that was set while sending the component in. And the date is not set so it is coming as null